model/textparse/openmetricsparse_test.go

```diff
@@ -992,3 +994,52 @@ go_gc_duration_seconds_created`)
 	require.Equal(t, "go_gc_duration_seconds", string(copyParser.l.b[copyParser.offsets[0]:copyParser.offsets[1]]))
 	require.False(t, copyParser.skipCTSeries)
 }
+
+func BenchmarkOMParse(b *testing.B) {
+	for parserName, parser := range map[string]func([]byte, *labels.SymbolTable) Parser{
+		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st)
+		},
+		"openmetrics-skip-ct": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st, WithOMParserCTSeriesSkipped())
+		},
+	} {
+		f, err := os.Open("omtestdata.txt")
+		require.NoError(b, err)
+		defer f.Close()
+
+		buf, err := io.ReadAll(f)
+		require.NoError(b, err)
+
+		b.Run(parserName+"/parse-ct/"+"omtestdata.txt", func(b *testing.B) {
+			if !strings.HasPrefix(parserName, "openmetrics") {
+				b.Skip()
+			}
```

Comment:
We don't need this now that this benchmark only covers OM, right?

Reply:
Could've sworn I got rid of this 😵

Reply:
Hmm looks like wasn't pushed for some reason